src/components/molecules/AppWrapper.tsx

```diff
@@ -13,9 +13,9 @@ interface AppWrapperProps {
 
 const AppWrapper = ({link, title, titleClassName = '', showBurger = false, children}: AppWrapperProps): JSX.Element => {
     return (
-        <div className="bg-page h-screen h-[100dvh]">
```

Comment:
There is a reason for `h-screen h-[100dvh]`. Dynamic viewport units are not supported in all browsers, so we first use the `h-screen` class to set the height to the device's screen, and then use the dynamic viewport height, which smartly calculates the height depending on the mobile phone's UI elements expanded/collapsed states. If unsupported, it does nothing. Could you please leave it like it ways?

Reply:
Nice! Back to normal now